### PR TITLE
Email field  validation on contact form

### PR DESCRIPTION
**Pull request by lmunozm1702**
## Implement the validation for the email field in lowercase.

The message must appear up to the field with a red background, as follow:

image

### CODE REVIEW

File: assets/js/index_contact.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const errorDiv = document.querySelector('#contact-email-error-message');
+
+  if (errorDiv) {
+    errorDiv.remove();
+  }
+
+  if (!validateEmail(form.elements['contact-email'])) {
+    setMessage('contact-email', 'Ooops!, please write your email in lowercase!', 'error');
+  } else {
+    form.submit();
```
Comment (KDlamini) [inline assets/js/index_contact.js]: Please make sure that the error message is near the submit button as per project requirements.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @lmunozm1702 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Email validation works as expected
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/index_contact.js (hunk 1 above)
Comment (lmunozm1702) [reply assets/js/index_contact.js]: It's done:

image

**Review by jerryowusu — APPROVED** [review]
Hi Team :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

wow approved


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Followed GitHub flow
:heavy_check_mark: Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**